taxonomies/additives.txt

```diff
@@ -11655,21 +11867,23 @@ wikipedia:en: https://en.wikipedia.org/wiki/Tara_spinosa#Uses
 #description:en:TARA GUM is produced by separating and grinding the endosperm of T. spinosa seeds. Tara gum is used as a thickening agent and stabilizer in a number of food applications.
 # ingredient/fr:gomme-tara has 254 products in 6 languages -@2019-05-12
 
-en: E418, Gellan gum, gellan
+en: E418, Gellan gum, gellan, E-418, E 418, INS418, INS-418, INS 418
```

Comment:
in theory the extra E-418, E 418, INS418, INS-418, INS 418 should not be needed, as we normalize the e-numbers when doing ingredients parsing

Reply:
In all languages, we have some preparsing code to change all variations like that to E418:

https://github.com/openfoodfacts/openfoodfacts-server/blob/main/lib/ProductOpener/Ingredients.pm#L6738